### ISSUE TITLE
The links in the contributor guide not working

### ISSUE THREAD

**Issue by soma2000-lang** [semi-technologies/weaviate-io#92]
[link](https://github.com/semi-technologies/weaviate-io/blob/main/developers/contributor-guide/current/weaviate-core/structure.html)

**Comment by Asmit2952** [comment]
It is working for me on the website: https://weaviate.io/developers/contributor-guide/current/weaviate-module-system/how-to-build-a-new-module.html

Scroll down to this [line](https://weaviate.io/developers/contributor-guide/current/weaviate-module-system/how-to-build-a-new-module.html#:~:text=If%20you%20want%20to%20make%20a%20pull%20request%20to%20Weaviate%20with%20your%20custom%20module%2C%20make%20sure%20to%20adhere%20to%20the%20code%20structure.)

**Comment by soma2000-lang** [comment]
Yes on the website its working but [link]{https://github.com/semi-technologies/weaviate-io/blob/main/developers/contributor-guide/current/weaviate-module-system/how-to-build-a-new-module.md}

**Comment by Asmit2952** [comment]
> Yes on the website it's working but here{[https://github.com/semi-technologies/weaviate-io/blob/main/developers/contributor-guide/current/weaviate-module-system/how-to-build-a-new-module.md}](https://github.com/semi-technologies/weaviate-io/blob/main/developers/contributor-guide/current/weaviate-module-system/how-to-build-a-new-module.md%7D) ie on the github page it is not

The site is made with Jekyll. It is a Static Site Generator (SSG). In short, what it does is, it generates HTML output from the .MD files. So, if you change any link in the .MD file, that link will be generated in HTML output, which will redirect it to wrong place. Hence, the link properly works on the site for now, because it is linked with the right page internally. 

If you want to fix links, you should go through file structure first, and see where the file is located and then link that file in the .MD file like this `(./folder-name/filename)`

**Comment by laura-ham** [comment]
Thanks @Asmit2952 for the explanation for @soma2000-lang, I hope this clarifies! I'll close this issue :)
